### DISCUSSION TITLE
How to add custom timezone on UTMSTACK

### DISCUSSION THREAD

**Discussion by chuilanzheng** [utmstack/UTMStack#765] [Q&A]
UTMSTACK is a great tool, unfortunately it is not complete for timezone, is there a way to add/custom timezone?

**Comment by osmontero** [comment]
Hello, UTMStack is an open source SIEM, feel free to contribute by adding the missing timezone and send us a pull request to integrate your changes. Before that, please open an issue using our issues template explaining your requirements, then you can link your pull request to that issue.
